README.md

```diff
@@ -38,34 +38,27 @@ Zim has many modules! Enable as many or as few as you'd like.
 Installation
 ------------
 Installing Zim is easy. If you have a different shell framework installed (like oh-my-zsh or prezto),
-*uninstall those first to prevent conflicts*. It can be installed manually by following the instructions below:
+**uninstall those first to prevent conflicts**. Then put following instructions at the end of your `~/.zshrc`:
 
-1. In a Zsh shell, clone the repository:
-  ```
-  git clone --recursive https://github.com/zimfw/zimfw.git ${ZDOTDIR:-${HOME}}/.zim
-  ```
+  ```bash
+  # Zim initializition
+  export ZIM_HOME="$HOME/.zim"
 
-2. Paste this into your terminal to prepend the initialization templates to your configs:
-  ```
-  setopt EXTENDED_GLOB
-  for template_file in ${ZDOTDIR:-${HOME}}/.zim/templates/*; do
-    user_file="${ZDOTDIR:-${HOME}}/.${template_file:t}"
-    touch ${user_file}
-    ( print -rn "$(<${template_file})$(<${user_file})" >! ${user_file} ) 2>/dev/null
-  done
-  ```
+  if [ ! -f "$ZIM_HOME/init.zsh" ]; then
+    echo "Installing zim"
+    git clone --recursive https://github.com/zimfw/zimfw.git $ZIM_HOME
+    cat $ZIM_HOME/templates/zlogin >> $HOME/.zlogin
```

Comment:
I'm asuming adding commands for installing zim to the zshrc is for users propagating settings to multiple machines. If that the case, why update .zlogin on every installation? The user may end up with multiple sourcings of login_init.zsh.

Reply:
First, the zim initialization steps inside the zshrc could definitely help to propagating settings.
`if [ ! -f "$ZIM_HOME/init.zsh" ];` would guarantee that sourcing login_init.zsh is only happened once.